framer/CanvasLayer.coffee

```diff
@@ -0,0 +1,29 @@
+{_} = require "./Underscore"
+
+{Layer} = require "./Layer"
+
+DPR = window.devicePixelRatio or 1
+
+class exports.CanvasLayer extends Layer
+	
+	constructor: (options={}) ->
+
+		super _.defaults options,
+			backgroundColor: null
+		
+		@_canvas = document.createElement("canvas")
+		@_ctx = @canvas.getContext("2d")
+		
+		@_update()
+		@_element.appendChild(@canvas)
+		@on("update:frame", @_update)
```

Comment:
You sure this shouldn't be `@on("change:frame", @_update)`?
At least this change (no pun) gives me the result I was expecting.